src/rbac/config/auth/authentication.py

```
@@ -24,3 +25,20 @@ def get_current_user(
                           role=AccountRole.from_str(payload.role))
     except JWTError:
         raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail="Invalid token")
+
+
+def get_current_user_from_request(request: Request, jwt_token_provider: JwtTokenProvider = Depends(Provide[Container.jwt_token_provider])) -> UserDetail:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)